README.md

```diff
@@ -34,31 +52,68 @@ Here is an example that would install the app to
 workload cluster `abc12`:
 
 ```
-# appCR.yaml
-
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: oms-agent
+  namespace: abc12
+spec:
+  catalog: giantswarm-playground
+  kubeConfig:
+    inCluster: false
+  name: oms-agent
+  namespace: oms-agent
+  version: 0.0.1
+  userConfig:
+    configMap:
+      name: oms-agent-user-values
+      namespace: abc12
+    secret:
+      name: oms-agent-user-secrets
+      namespace: abc12
 ```
 
 ```
-# user-values-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oms-agent-user-values
+  namespace: abc12
+data:
+  values: |
+    env:
+      clusterName: w5fc9 
 
+```
 
 ```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: oms-agent-user-secrets
+  namespace: abc12
+data:
+  omsagent:
+    secret:
+      wsid: Azure_Log_Analytics_Workspace_ID
+      key: Azure_Log_Analytics_Primary_Key
+```
 
 See our [full reference page on how to configure applications](https://docs.giantswarm.io/app-platform/app-configuration/) for more details.
 
 ## Compatibility
 
 This app has been tested to work with the following workload cluster release versions:
 
-*
+* Azure v15.0.1
 
 ## Limitations
 
 Some apps have restrictions on how they can be deployed.
 Not following these limitations will most likely result in a broken deployment.
 
-*
+* Log Analytics Workspace ID and Key is a requirement.
 
 ## Credit
 
-* {APP HELM REPOSITORY}
+* https://github.com/microsoft/OMS-Agent-for-Linux
```

Comment:
The chart is from here, right?

https://github.com/microsoft/Docker-Provider/tree/ci_prod/charts/azuremonitor-container

Reply:
yes yes those are the 'releases' of MS

Reply:
fixed